src/views/News/index.jsx

```diff
@@ -259,7 +241,7 @@ const StudentNews = () => {
       createSnackbar('News Posting Updated Successfully', 'success');
     }
 
-    setOpenPostActivity(false);
+    handleWindowClose();
```

Comment:
Did you read my note on the last comment here? I don't think you want to be clearing out the form if the update failed.

Reply:
My bad. Now it should only clear out the form when the update is a success.